### PR TITLE
[DOCS] Bulk Editing Rules: Timeline Template

### LINKED ISSUE #1816: Bulk Editing Rules: Timeline Template
## Description

Users can apply a timeline template to multiple rules at the same time, using the **Bulk actions** menu in rules table.

Related: 
* Issue: https://github.com/elastic/security-team/issues/2078
* PRs: 
   * https://github.com/elastic/kibana/pull/128691
   * https://github.com/elastic/kibana/pull/129491

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #1816.

Preview: [Manage detection rules | Modify existing rules settings](https://security-docs_1842.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-management.html#edit-rules-settings)

### CODE REVIEW

File: docs/detections/rules-ui-manage.asciidoc [hunk 1]
```diff
@@ -73,16 +73,17 @@ image::images/install-prebuilt-rules.png[]
...
 NOTE: For prebuilt Elastic rules, you can't modify most settings. You can only edit <<rule-schedule, rule actions>> and <<detections-ui-exceptions, add exceptions>>.
 
 . Go to *Detect* -> *Rules*.
 . Do one of the following:
 * Edit a single rule: Select the *All actions* menu (*...*) on a rule, then select *Edit rule settings*. The *Edit rule settings* view opens, where you can modify the <<rules-ui-create, rule's settings>>.
-* Edit multiple rules: Select all the rules you want to edit, then select an action from the *Bulk actions* menu:
+* Bulk edit multiple rules: Select the rules you want to edit, then select an action from the *Bulk actions* menu:
 ** *Index patterns*: Add or delete the index patterns used by all selected rules.
 ** *Tags*: Add or delete tags on all selected rules.
+** *Apply Timeline template*: Apply a specified <<timeline-templates-ui, Timeline template>> to selected rules. You can also choose *None* to remove Timeline templates from the selected rules.
```
Comment (benironside) [inline docs/detections/rules-ui-manage.asciidoc:86]:
```suggestion
** *Apply Timeline template*: Apply a specified <<timeline-templates-ui, Timeline template>> to the selected rules. You can also choose *None* to remove Timeline templates from the selected rules.
```
Matching the next sentence ("to the", "from the")

**Review by benironside — COMMENTED** [review]
lgtm, left one minor suggestion

**Review by banderror — APPROVED** [review]
LGTM 👍

**Review by jethr0null — APPROVED** [review]
LGTM!

**Review by jmikell821 — APPROVED** [review]
LGTM!

**Comment by prachigupta-qasource** [comment]
Hi @joepeeples ,

We tested the linked docs and found that all the docs are correctly updated as per the latest UI. So we are good to go ahead and merge the changes.

Marking this ticket as 'QA validated'.

Thanks!

**Pull request merged**